app/models/grda_warehouse/cas_project_client_calculator/boston.rb

```diff
@@ -169,10 +169,10 @@ def value_for_cas_project_client(client:, column:)
     end
 
     private def default_shelter_agency_contacts(client)
-      # TODO: this is being moved to custom pull from Clarity
+      client.client_contacts.shelter_agency_contacts.where.not(email: nil).pluck(:email)
+    end
 
-      # client.most_recent_pathways_or_rrh_assessment_for_destination.
-      #   question_matching_requirement('c_casemanager_contacts')&.AssessmentAnswer
+    private def contact_info_for_rrh_assessment(client)
```

Comment:
Should this be empty? Maybe a comment to note why we don't have anything to return?

Reply:
Nope, it should now have the appropriate code.